app/partials/app.cjsx

```diff
@@ -12,35 +13,53 @@ PanoptesApp = React.createClass
     initialLoadComplete: React.PropTypes.bool
     user: React.PropTypes.object
     geordi: React.PropTypes.object
+    notificationsCounter: React.PropTypes.object
+    unreadNotificationsCount: React.PropTypes.number
 
   getChildContext: ->
     initialLoadComplete: @state.initialLoadComplete
     user: @state.user
     geordi: @geordiLogger
+    notificationsCounter: @state.notificationsCounter
+    unreadNotificationsCount: @state.unreadNotificationsCount
 
   getInitialState: ->
     initialLoadComplete: false
     user: null
+    notificationsCounter: new NotificationsCounter()
```

Comment:
Yep, the counter changed to a prop in a70a3aba.  The unread count is still a state, but that shouldn't need to change.